cmd/changelog-parser/main.go

```diff
@@ -247,6 +253,8 @@ func main() {
 		"Output filename")
 	flag.StringVar(&version, "v", defaultVersionString,
 		"Version to embed in the changelog")
+	flag.StringVar(&apiToken, "p", "",
```

Comment:
Just curious, did you consider `a` or `T` or `g` for the GitHub API token flag (vs. "p")?

Reply:
- `T` - generally, I think most tools try to stay with the same casing as the rest of the options so I tried to avoid overloading "t"
- `a` + `g` didn't seem as close to the meaning as a `p` since it's technically the password for the API